client/store/actions.js

```diff
@@ -1,2 +1,55 @@
 // Store all action used in project
-export default {}
\ No newline at end of file
+import axios from 'axios';
+import * as mutations from './mutations';
+
+export default {
+    userLogin({commit}, payload) {
+        const { username, password } = payload
+
+        axios.post('/rest-auth/login/', {
+            username: username,
+            password: password,
+        })
+        .then((response) => {
+            const { data: {
+                token
```

Comment:
I think you should also save user object to store. I think you should send data to the mutation and with object destruction get user and token or make both migrations idk.

BTW why can't you do something like this:
```
Migrations:
setToken(state, { token }) {...}
```